### DISCUSSION TITLE
question on Alt-Y

### DISCUSSION THREAD

**Discussion by firolightfog** [schismtracker/schismtracker#444] [Q&A] [answered]
What is Alt-Y in the Sample List (F3) page exactly?!
Many thanks, Andras

**Comment by jangler** [comment, marked as answer]
Not a super useful feature IMO, but it lets you enter text which it converts to 8-bit sample data based on the encoding (code page 437 extended ASCII?) value of the characters entered.

**Comment by esaruoho** [comment]
hey this is super useful for chiptune music

**Comment by firolightfog** [comment]
So is it like loading a TXT file insted of a WAV into the Sample List?

**Comment by jangler** [comment]
Basically, yes.
